### ISSUE TITLE
False positive warning about paths overridden by provided as command arguments

### ISSUE THREAD

**Issue by kubawerlos** [FriendsOfPHP/PHP-CS-Fixer#3044]
In latest 2.6.0 tag for a config:
```php
<?php

return PhpCsFixer\Config::create();
```

when running:
```bash
vendor/bin/php-cs-fixer fix --config=config-from-above.php src
```

I'm getting "Paths from configuration file have been overridden by paths provided as command arguments" warning.

**Comment by julienfalque** [comment]
This looks expected: [the default finder](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.5/src/Finder.php#L27) will iterate all .php/.phpt files in the current directory, excluding vendor, dot files and VCS files. So when you provide `src` as path argument, you do override this finder.

**Comment by kubawerlos** [comment]
Seems misleading at least. I'd expect the warning when I define custom finder in configuration and override on command line.

**Comment by keradus** [comment]
@kubawerlos 
just to be extra sure, could you show your config file please?

**Comment by kubawerlos** [comment]
@keradus spotted this with updating [this](https://github.com/kubawerlos/grid/blob/develop/tests/php-cs-fixer.config.php), but also happened with the one-liner from first post.

**Comment by keradus** [comment]
@julienfalque 
default finder has some filtering, but is not assuming the place to search in.

if you use default finder without manually configuring the path in config and not providing the path for it as CLI arg, then we got error:
> You must call one of in() or append() methods before iterating over a Finder.

if one has not configured path in config file and then configure it on CLI level, he shall not see the warning indeed.


@kubawerlos , are you willing to send a fix for it ? (for 2.2 line)

**Comment by kubawerlos** [comment]
@keradus yes, I will
